.github/workflows/npm-workflow.yml

```diff
@@ -120,6 +124,31 @@ jobs:
       if: ${{ failure() }}
       run: cat /github/home/.npm/_logs/*.log
 
+    - name: Run lint-staged
+      if: ${{ inputs.run_lint_staged == 'true' }}
+      run: |
+        # stash changes done by bumping version
+        git stash
+        echo ""
```

Comment:
what are those `echo ""` ?

Reply:
Just there to help make the output more readable, see https://github.com/MOV-AI/frontend-npm-lib-scene/actions/runs/10593728006/job/29355866434#step:12:176